src/conn/conn_api.c

```diff
@@ -1107,13 +1107,18 @@ __conn_close(WT_CONNECTION *wt_conn, const char *config)
     WT_DECL_RET;
     WT_SESSION *wt_session;
     WT_SESSION_IMPL *s, *session;
+    WT_TIMER timer;
+    uint64_t time_diff;
     uint32_t i;
 
     conn = (WT_CONNECTION_IMPL *)wt_conn;
 
     CONNECTION_API_CALL(conn, session, close, config, cfg);
 err:
 
+    /* Initialize the tracking timer */
```

Comment:
Nitpick: missing full stop, most of the other "initialise the timer" comments are the same.

Reply:
Do we need a comment any more? It seems pretty obvious from the function name.

Reply:
I went ahead and removed those comments as it is understandable from the function name itself.